native/cocos/editor-support/spine-creator-support/SpineAnimation.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include <spine/Animation.h>
+#include <vector>
+#include <memory>
+
+namespace spine {
+
+class SpineAnimation : public Animation {
+public:
+    SpineAnimation(const String &name, std::vector<std::shared_ptr<Timeline>> timelines, float duration);
+
+    ~SpineAnimation();
+
+private:
+    std::vector<std::shared_ptr<Timeline>> _vecTimelines;
+    spine::Vector<Timeline*> _spineVecTimelines;
```

Comment:
Why need to use two different containers at the same time?

Reply:
spine::Animation's constructor requires spine::Vector<Timeline*>, we need to change from std::vector to spine::Vector